app/controllers/hello_controller.rb

```diff
@@ -2,6 +2,7 @@ class HelloController < ApplicationController
   def index
     @picture = current_user.picture_url
     @user_name = current_user.full_name
-    @posts = Post.all
+    posts = Post.all.sort_by { |post| post[:created_at] }
+    @recent_posts = posts.reverse[0..4]
```

Comment:
We thought that the homepage should show only the 5 most recent posts. Sound good?